webui/src/components/reportabuse.jsx

```diff
@@ -0,0 +1,379 @@
+
+                        <div className='row' style={gap}>
+                            <div className="col-sm-4">
+                                <label forHtml='name'>* Your full name </label>
+                            </div>
+                            <div className=" col-sm-8" >
+                                <input ref={(ref) => this.name = ref} name='name' className="form-control" type='text'  required />
+                            </div>
+                        </div>
+
+                        <div className='row' style={gap}>
+                            <div className="col-sm-4" >
+                                <label forHtml='affiliation'>* Your affiliation </label>
+                            </div>
+                            <div className=" col-sm-8" >
+                                <input ref={(ref) => this.affiliation = ref}  name='affiliation' className="form-control" type='text' required />
+                            </div>
+                        </div>
+
+                        <div className='row' style={gap}>
+                            <div className="col-sm-4">
+                                <label forHtml='email'>* Your email </label>
+                            </div>
+                            <div className=" col-sm-8" >
+                                <input ref={(ref) => this.email = ref} name='email' className="form-control" type='email' required />
+                            </div>
+                        </div>
+
+                        <div className='row'  style={gap}>
+                            <div className="col-sm-4">
+                                <label forHtml='address'>* Your address </label>
+                            </div>
+                            <div className=" col-sm-8" >
+                                <input ref={(ref) => this.address = ref} name='address' className="form-control" type='text'  required />
+                            </div>
+                        </div>
+
+                        <div className='row'  style={gap}>
+                            <div className="col-sm-4">
+                                <label forHtml='city'>* City </label>
+                            </div>
+                            <div className=" col-sm-8" >
+                                <input ref={(ref) => this.city = ref} name='city' className="form-control" type='text'  required />
+                            </div>
+                        </div>
+
+                        <div className='row'  style={gap}>
+                            <div className="col-sm-4">
+                                <label forHtml='country'>* Country </label>
+                            </div>
+                            <div className=" col-sm-8" >
+                            <select ref={(ref) => this.country = ref} name='country' className="form-control" defaultValue=""  required >
+                                <option value=""> Choose country </option>
```

Comment:
The list of countries should be put in an array and expanded later, it takes too much space and is doubled between /abuse and /accessrequests